.github/workflows/continous-integration.yml

```diff
@@ -352,9 +352,13 @@ jobs:
           (Get-ItemProperty "HKLM:System\CurrentControlSet\Control\FileSystem").LongPathsEnabled
           Set-ItemProperty 'HKLM:\System\CurrentControlSet\Control\FileSystem' -Name 'LongPathsEnabled' -value 0
 
-      - name: Install ddtrace
-        if: needs.changes.outputs.backend == 'true'
-        run: poetry run pip install -U ddtrace
+      - name: Install ddtrace on Linux
+        if: needs.changes.outputs.backend == 'true' && matrix.os == 'ubuntu-22.04'
+        run: poetry run pip install -U 'ddtrace<2.0.0'
+
+      - name: Install ddtrace on Windows
+        if: needs.changes.outputs.backend == 'true' && matrix.os == 'windows-2019'
+        run: py -m pip install -U 'ddtrace<2.0.0'
```

Comment:
@miraai Is there a unified command provided by Github actions that allows us to install a dependency with a constraint for both OS types?

Reply:
@ancalita We can use `RUNNER_OS` envar to put this into a single step, your call though if it is worth it

```
- name:  Install ddtrace
  if: needs.changes.outputs.backend == 'true'
  run:   |
         if [ "$RUNNER_OS" == "Linux" ]; then
              poetry run pip install -U 'ddtrace<2.0.0'
         elif [ "$RUNNER_OS" == "Windows" ]; then
              py -m pip install -U 'ddtrace<2.0.0'
         else
              echo "$RUNNER_OS not supported"
              exit 1
         fi
```

Reply:
This is documented here, https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables